frontend/src/components/Worksheet/WorksheetStats.tsx

```diff
@@ -120,7 +120,7 @@ export default function WorksheetStats() {
     // const compressed = await lzma.compress(wsSerial, 9);
     // const binaryCompressed = Array.from(compressed, (byte) => String.fromCodePoint(byte+128)).join("")
     await navigator.clipboard.writeText(
-      `https://localhost:3000/worksheet?ws=${compressToEncodedURIComponent(wsSerial)}`,
+      `${window.location.origin}/worksheet?ws=${compressToEncodedURIComponent(wsSerial)}`,
```

Comment:
not harcoding so we can use in prev envs etc unless @Josh-Cena thinks this isnt safe or efficient for some reason